code/modules/mob/living/carbon/superior_animal/giant_spider/attack.dm

```diff
@@ -0,0 +1,7 @@
+/mob/living/carbon/superior_animal/giant_spider/UnarmedAttack(var/atom/A, var/proximity)
```

Comment:
Is this proc called on humans? This wouldn't work for them (or maybe all carbon mobs) since they have multiple reagent holders. Chems need to be injected into bloodstr

Reply:
This is a proc that use used for unarmed attacks by player controlled mobs afaik. What I did was move all special attack info into this and had the AI utilize it for attacks instead.

Reply:
but it contains a call just a few lines down ```L.reagents.add_reagent(poison_type, poison_per_bite)```

Who is that intended for? I don't see how it could function correctly in any circumstance